example/src/main/java/ably/io/tutorial/ExampleActivity.java

```diff
@@ -75,4 +93,14 @@ private void publishMessages() throws AblyException {
         }
     }
 
+    private void fetchHistory() throws AblyException {
+        Log.v(getClass().getCanonicalName(), "Retrieving channel's history...");
+
+        /* Fetch historical messages from channel, you can customize history query with parameters, when no parameters are needed just pass null */
+        Message[] historicMessages = channel.history(null).items();
+        for (Message message : historicMessages) {
+            Log.v(getClass().getCanonicalName(), "message: " + message.id + " - " + message.data);
```

Comment:
This seems odd that interacting with a UI provides no feedback and instead logs messages to a console users cannot see.  We need to output messages to a UI.